test/surface/test_poisson.cpp

```diff
@@ -67,6 +67,9 @@ TEST (PCL, Poisson)
 
   Poisson<PointNormal> poisson;
   poisson.setInputCloud (cloud_with_normals);
+  // poisson4 returns different vertices when parallelized
```

Comment:
Perhaps it makes sense then to keep the "default is no parallelization" for `Poisson`? That is, to set `num_threads_=1` in the `Poisson`  constructor?